website/content/docs/sync/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+page_title: Secrets Sync
+description: Secrets sync allows you to safely sync Vault-managed secrets with external destinations.
+---
+
```

Comment:
We can add an [Enterprise alert](https://github.com/hashicorp/vault/tree/docs/saml-auth-method/website#enterprise-alert) here. It will automatically render with the Vault logo.

I think it's also common on the top of the API docs page for ent features.

Reply:
Yup done!